paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -363,6 +365,24 @@ def can_user_deploy_service(deploy_info: Dict[str, Any], service: str) -> bool:
     return True
 
 
+def can_run_metric_watcher_threads(
+    service: str,
+    soa_dir: str,
+) -> Union[bool, List[str]]:
+    """
+    Cannot run slo and metric watcher threads together for now.
+    SLO Watcher Threads take precedence over metric watcher threads.
+    Metric Watcher Threads can run if there are no SLOs available.
+    """
+    slo_files = get_files_of_type_in_dir(
+        file_type="slo", service=service, soa_dir=soa_dir
+    )
+    rollback_files = get_files_of_type_in_dir(
+        file_type="rollback", service=service, soa_dir=soa_dir
+    )
+    return not slo_files and rollback_files
```

Comment:
oh, sorry: i think we'll want this to be 
```suggestion
    return bool(not slo_files and rollback_files)
```

so that this will always return a boolean (and thus avoid the need to type this as returning a union)